src/main/java/org/reaktivity/command/log/internal/LoggableRoutes.java

```diff
@@ -106,14 +107,45 @@ private int logRoutes(
             if (!loggedRoutes.contains(correlationId))
             {
                 workCnt.incrementAndGet();
-                out.printf(format("%15s   %-10s %-20s [0x%016X] %-20s [0x%016X] [0x%016X]\n",
-                                format("%s#%d", nukleusName, correlationId),
-                                role,
-                                source,
-                                sourceRef,
-                                target,
-                                targetRef,
-                                authorization));
+
+                String extension = null;
+                if ("tls".equals(nukleusName))
+                {
+                    TlsRouteExFW ext = new TlsRouteExFW();
+                    final int index = route.extension().offset();
+                    ext.wrap(route.extension().buffer(), index, index + route.extension().sizeof());
+                    final String applicationProtocol = ext.applicationProtocol().asString();
+                    final String hostname = ext.hostname().asString();
+                    final String store = ext.store().asString();
+                    extension = String.format(
+                    "{" +
+                    "\"store\":\"%s\"," +
+                    "\"hostname\":\"%s\"," +
+                    "\"applicationProtocol\":\"%s\"" +
+                    "}",
+                    hostname,
+                    applicationProtocol,
+                    store);
+                }
```

Comment:
Hoist this to another method returning `extension`, where we can later add support for other nuklei.